NEWS.md

```diff
@@ -22,32 +22,94 @@ OpenSSL 3.2
 
 ### Major changes between OpenSSL 3.1 and OpenSSL 3.2 [under development]
```

Comment:
Is is still supposed to say [under development] when you are about to release?

Reply:
Yes.  The release script will change this automatically for the release commit (the one that's tagged with the release tag), and will change it back on the followup commit.  You can see that in action if you study, for example, 01d5e2318405362b4de5e670c90d9b40a351d053 (release of 3.1.4) and its followup 7b2163f485c51777cc812eb967076c1357ad663b (update to 3.1.5-dev).